c/pthread/README.txt

```diff
@@ -1 +1,2 @@
-Contributed by: ESBMC project
+Contributed by: ESBMC project (unknown license) and Sosy-Lab (Apache2)
```

Comment:
Now one has to look up in the commit history which tasks are under Apache2 license. Would it be worthwhile having a small license header in each of the new tasks that are published under the Apache2 license?